### PR TITLE
[WFLY-17394] Update WildFly Faces docs for Multi JSF Support

### PR DESCRIPTION
**Pull request by jasondlee**
Add missing doc file
Add section discussing the wildfly-extras feature pack

### CODE REVIEW

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Server_Faces.adoc [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  </config>
+  <options>
+    <option name="optional-packages" value="passive+"/>
+    <option name="jboss-fork-embedded" value="true"/>
+  </options>
+</installation>
+-----
+
+[[manually-installing-a-new-jsf-implementation]]
+== Manually installing a new Jakarta Server Faces implementation
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Server_Faces.adoc:59]: I feel a bit doubtful about documenting this. I don't think we actually want anyone to do this.

Comment (jasondlee) [reply docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Server_Faces.adoc:59]: > I feel a bit doubtful about documenting this. I don't think we actually want anyone to do this.

I think I agree. IIRC, it's re-addition was explicitly requested, but the feature pack changes things a bit. If someone wants to do this manually, the FP should show how to do that well enough that we need not document that here.

File: docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Faces.adoc [hunk 2]
```diff
@@ -0,0 +1,122 @@
+:module-url: https://raw.githubusercontent.com/wildfly/wildfly/main/jsf/multi-jsf-installer/src/main/resources
+
+[[Jakarta_Faces]]
+= Jakarta Faces Configuration
+
+[abstract]
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Faces.adoc:6]: @jasondlee This tag results in the paragraph being rendered in html in italics, which is funky looking and out of sync with the rest of the Admin Guide. Can your remove this please?

Comment (jasondlee) [reply docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Faces.adoc]: @bstansberry Done!

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @jasondlee

**Pull request merged**